com.github.iwalton3.jellyfin-media-player.json

```
@@ -0,0 +1,205 @@
+{
+  "app-id": "com.github.iwalton3.jellyfin-media-player",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.12",
```

Update runtime to 5.15
```suggestion
  "runtime-version": "5.15",
```